src/mlt++/MltProperties.cpp

```diff
@@ -371,6 +371,16 @@ int Properties::set( const char *name, mlt_color value )
 	return mlt_properties_set_color( get_properties(), name, value );
 }
 
+mlt_color Properties::anim_get_color(const char *name, int position, int length)
```

Comment:
I would really love to see unit tests for these functions here:
https://github.com/mltframework/mlt/blob/a7ecae88bf23014b9359ca9f792670efcf3d1942/src/tests/test_properties/test_properties.cpp#L165

But I am not sure if that should be required for this request. @jlskuz do you have any interest in trying to add unit tests? It is quite easy with the test framework we have. But if you are not interested, I can withdraw this comment.

Reply:
I think it makes sense to have unit tests for this. Will try to write some.

Reply:
I think I covered everything related to animated colors with tests now.